### PR TITLE
Fix strict include compile error

### PR DESCRIPTION
**Pull request by dorgonman**
Hi, this pull request fix compile error when compile plugin using following command:

`RunUAT.bat BuildPlugin -StrictIncludes -Plugin="/Path/To/Plugins/SaveExtension/SaveExtension.uplugin" -TargetPlatforms="Win64" -nop4 -ue4exe="/Path/To/Engine/Binaries/Win64/UE4Editor-Cmd.exe" -Package="/Path/To/PackagePlugin/Plugin/Win64/feature/SaveGame/"`

### CODE REVIEW

File: Source/SaveExtension/Private/Serialization/SlotDataTask_Loader.cpp [hunk 1]
```diff
@@ -4,12 +4,16 @@
...
+#include <Kismet/GameplayStatics.h>
+#include <Components/PrimitiveComponent.h>
 
 #include "SavePreset.h"
 #include "SaveManager.h"
 #include "Serialization/SEArchive.h"
 
 
+
+
```
Comment (muit) [inline Source/SaveExtension/Private/Serialization/SlotDataTask_Loader.cpp]: Maybe remove this extra lines?

File: Source/SaveExtension/Public/Serialization/MTTask_SerializeActors.h [hunk 2]
```diff
@@ -12,10 +12,14 @@
...
 
 #include "MTTask.h"
+#include "Serialization/Records.h"
+#include "Serialization/LevelRecords.h"
+
 
 
 class USlotData;
 
+
```
Comment (muit) [inline Source/SaveExtension/Public/Serialization/MTTask_SerializeActors.h]: Extra lines?

File: Source/SaveExtension/Private/Serialization/MTTask_SerializeActors.cpp [hunk 3]
```diff
@@ -10,6 +11,7 @@
 #include "Serialization/SEArchive.h"
 
 
+
```
Comment (muit) [inline Source/SaveExtension/Private/Serialization/MTTask_SerializeActors.cpp]: Extra lines?

File: Source/SaveExtension/Private/Serialization/SlotDataTask_Loader.cpp (hunk 1 above)
Comment (dorgonman) [reply Source/SaveExtension/Private/Serialization/SlotDataTask_Loader.cpp]: done

File: Source/SaveExtension/Public/Serialization/MTTask_SerializeActors.h (hunk 2 above)
Comment (dorgonman) [reply Source/SaveExtension/Public/Serialization/MTTask_SerializeActors.h]: done

File: Source/SaveExtension/Private/Serialization/MTTask_SerializeActors.cpp (hunk 3 above)
Comment (dorgonman) [reply Source/SaveExtension/Private/Serialization/MTTask_SerializeActors.cpp]: done

**Comment by muit** [comment]
Thank you :)

**Pull request merged**